resolutions/-resolution-annual-xxx.md

```diff
@@ -0,0 +1,32 @@
+# ANNUAL RESOLUTIONS OF THE BOARD OF DIRECTORS
```

Comment:
I imagine this is based on https://github.com/hyphacoop/organizing/blob/master/resolutions/resolution-001.md do we need the word `ANNUAL`? Is it to differentiate it as SPECIAL? If so we should keep two templates?

Reply:
Yes, this was for a bunch of resolutions at the annual meeting -- I'm not sure if it matters tbh :woman_shrugging:

Reply:
I figured we could start with this, and then modify to make a new template if/when we need our next one-off resolution!